skeleton_scary_game.py

```diff
@@ -0,0 +1,151 @@
+            item_path = random.choice(item_paths)
+    
+            # This is where the lore is
+            lore_paths = [path for path in self.paths if (path != demon_path and path not in item_path)]
+            lore_path = random.choice(lore_paths)
+            
+            
+
+            #Ask the user if they want to use their flashlight to see where the demon is.
+            need_help = input(str("Would you like to use your flashlight? (y/n) -> "))
+
+            # If the user picks an option that is not y or n, they have to repick.
+            if need_help.lower() != "y" and need_help.lower() != "n":
+                print("Pick yes or no")
+                continue
+
+
+            # If the user picks 'y' and they have a battery, it will call the father class and execute the use_item function. -> brings in users_bagpack from the item class and demon_path
+            # This is because the 'father' needs to check if he has batteries or not, and if he does, the demon path will be known. This will be executed in the father class.
+            if need_help.lower() == "y":
+                if "Battery" in item.users_bagpack:
+                    self.father.use_item(self.item.users_bagpack,demon_path)
+                else:
+                    print("You don't have any batteries \n")
+                    pass
+            elif need_help.lower() == "n":
+                pass
+
+            # Display the available paths
+            for i, path in enumerate(self.paths):
+                print(f"{i + 1}. {path}\n")
+            
+            # User picks their path
+            loop_count += 1
+            user_choice = int(input(f"Tunnel #{loop_count} "
+                                            f"Walk your path => "))
+
+
+            # If the user picks a path number that is not valid, they have to re-pick
+            if user_choice < 1 or user_choice > 3:
+                print("Invalid choice! Please enter a valid path number.")
+                continue
+
+            # Evaluate the chosen path
+            chosen_path = self.paths[user_choice - 1]
+
+            # If the user picks this path, then it will call the demon class
+            if chosen_path == demon_path:
+                self.demon.death()
+            
+            # If the user picks this path it will call the item class.
+            elif chosen_path == item_path:
+                self.item.collect_item()
+            
+            # If the user picks this path then nothing happens but just run the loop again.
+            elif chosen_path == lore_path:
+                self.lore.lore_play()
+            
+        # Once the user has survived 7 rounds then they win the game and get their daugther back.
+        if loop_count >= 7:
+            print("The demon said thank you for playing and returned your child")
+            exit(0)
+
+# The father class, and the only thing this class can do is use and item.
```

Comment:
This comment should be moved down into the class. See what it says here: https://android.googlesource.com/platform/external/google-styleguide/+/refs/tags/android-s-beta-2/pyguide.md#3_8_4-classes

See this comment there about the docstring: `Classes should have a docstring below the class definition describing the class. If your class has public attributes, they should be documented here in an Attributes section and follow the same formatting as a [function's Args](https://android.googlesource.com/platform/external/google-styleguide/+/refs/tags/android-s-beta-2/pyguide.md#doc-function-args) section.`